kld/tests/smoke/start.rs

```diff
@@ -181,8 +182,9 @@ pub async fn test_start() -> Result<()> {
     Ok(())
 }
 
+#[serial_test::serial]
```

Comment:
There are no circumstance where we would need to run both of these tests together so we don't need the serial crate.

Reply:
`cargo test`.  Nevermind, this is removed.